ovs/dal/tests/test_alba.py

```diff
@@ -112,87 +112,124 @@ def test_localstack(self):
         ASDManagerClient.test_exceptions[node] = {}
         ASDManagerClient.test_results[node] = {'get_stack': {},
                                                'get_metadata': {'_version': 3}}
-        expected = {'node_1': {}}
+        expected = {}
         node.invalidate_dynamics()
-        self.assertDictEqual(alba_backend._local_stack(), expected)
+        self.assertDictEqual(node._stack(), expected)
 
         # Validate local_stack when the node is offline
         ASDManagerClient.test_exceptions[node] = {'get_stack': requests.ConnectionError('test'),
                                                   'get_metadata': requests.ConnectionError('test')}
-        ASDManagerClient.test_results[node] = {'get_stack': {'alba_disk_1': {'available': True,
-                                                                             'device': '/dev/disk/by-id/alba_disk_1',
-                                                                             'name': 'alba_disk_1',
-                                                                             'osds': {},
-                                                                             'node_id': node.node_id,
-                                                                             'state': 'ok'}},
-                                               'get_metadata': {'_version': 3}}
-        expected = {'node_1': {}}
+        ASDManagerClient.test_results[node] = {'get_stack': {},
+                                               'get_metadata': {}}
+        expected = {}
         node.invalidate_dynamics()
-        self.assertDictEqual(alba_backend._local_stack(), expected)
+        self.assertDictEqual(node._stack(), expected)
 
         # Validate uninitialized disks
+        def _move(info):
+            for move in [('state', 'status'),
+                         ('state_detail', 'status_detail')]:
+                if move[0] in info:
+                    info[move[1]] = info.pop(move[0])
         ASDManagerClient.test_exceptions[node] = {}
-        expected = {'node_1': {'alba_disk_1': {'available': True,
-                                               'device': '/dev/disk/by-id/alba_disk_1',
-                                               'name': 'alba_disk_1',
-                                               'node_id': 'node_1',
-                                               'osds': {},
-                                               'status': 'ok',
-                                               'status_detail': 'unknown'}}}
+        get_stack = {'get_stack': {'alba_slot_1': {'aliases': ['/dev/disk/by-path/pci-0000:00:01.0-ata-1'],
```

Comment:
It's just a mocked piece of data that the asd-manager would return. Not skipping any keys.